### DISCUSSION TITLE
404 error, index.html is in /dist

### DISCUSSION THREAD

**Discussion by kiwi-lifter** [community/community#23026] [Other Feature Feedback, Questions, & Ideas]
<p>I get a 404 on my GitHub page built from the master branch. It says the index.html is missing. My project has a /dist subfolder which has the index.html. Is this why I get a 404? If so, is there a work around? Thanks in advance.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Github pages can only be published from master, gh-pages branch, or /docs.</p>
<p>Try creating a branch called gh-pages using your /dist folder, using <a href="https://help.github.com/en/articles/splitting-a-subfolder-out-into-a-new-repository" rel="nofollow noopener">this article</a>.</p>
